src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/BamboraConfiguration.java

```diff
@@ -18,8 +21,19 @@
 
 	@RequestMapping(value = "/bamboraconfiguration", method = RequestMethod.GET)
 	public BeanstreamEndpointResponse singlepaymenturl() {
-		return new BeanstreamEndpointResponse(approved, declined, error,
-				PaymentServiceConstants.PAYMENT_SERVICE_RESP_MSG_OK,
-				PaymentServiceConstants.PAYMENT_SERVICE_SUCCESS_CD);
+		try {
+			// Make sure autowired params are not null (May be missing from configMap)
+			if ( StringUtils.isEmpty(approved) || StringUtils.isEmpty(declined) || StringUtils.isEmpty(error))
+				throw new PaymentServiceException(PaymentServiceConstants.PAYMENT_SERVICE_ERR_MISSING_CONFIG_PARAMS);
+			return new BeanstreamEndpointResponse(approved, declined, error,
+					PaymentServiceConstants.PAYMENT_SERVICE_RESP_MSG_OK,
+					PaymentServiceConstants.PAYMENT_SERVICE_SUCCESS_CD);
+		}
+		catch (PaymentServiceException e)
+		{
+			e.printStackTrace();
```

Comment:
we should not use this, but use the logger to log the fatal error

Reply:
removed e.printstackTrace() . Logging will be added once the log4j dependency is merged to master